<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CCustomer</class>
 <widget class="QWidget" name="CCustomer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>212</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="hLayoutUnionViews">
     <item>
      <layout class="QVBoxLayout" name="vLayoutUnionPartnerHuman">
       <property name="spacing">
        <number>0</number>
       </property>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="hLayoutSearchToItem"/>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QHBoxLayout" name="hLayoutInformation">
     <item alignment="Qt::AlignLeft|Qt::AlignBottom">
      <widget class="QLabel" name="labelCurrentUser">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignRight|Qt::AlignBottom">
      <widget class="QLabel" name="labelViewState">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
